Documentation/GeneralConventions/DirectoryFilenames.rst

```diff
@@ -3,35 +3,30 @@
 
 .. _general-conventions-dir-and-filenames:
 
-=====================
-Directories and Files
-=====================
+==========================
+Directories and File Names
+==========================
 
-Directory and File Names
-========================
 
 * All documentation resides in a :file:`Documentation` subfolder
 * This folder contains the files :file:`Settings.cfg`, :file:`Includes.txt` and :file:`Index.rst`
-* Use **CamelCase** for file and directory names.
 * reSt files have ending **.rst**
 * Included files have ending **.rst.txt**
-
+* The :ref:`composer-json` file is valid and contains `typo3/cms-core` in the `required` section.
```

Comment:
should this be composer.json (dot instead of dash)

Reply:
This is a cross-reference. The title of the section with the label "composer-json" should be rendered, which is "composer.json". But I will check to make sure.

Reply:
Rendered page looks good: https://docs.typo3.org/typo3cms/HowToDocument/GeneralConventions/DirectoryFilenames.html